Makefile

```diff
@@ -55,18 +43,19 @@ test:
 	#     - Our Visual Studio Marketplace link for the Azure Pipelines task extension,
 	#       although valid and publicly available, is reported as a broken link.
 	# Fixes for the former two are tracked by https://github.com/pulumi/docs/issues/568.
-	./node_modules/.bin/blc http://localhost:4000 -r \
+	./node_modules/.bin/blc http://localhost:1313 -r \
 		--exclude "/reference/pkg" \
-		--exclude "/reference/changelog.html" \
+		--exclude "/reference/changelog" \
 		--exclude "https://github.com/pulumi/docs/edit/master" \
 		--exclude "https://www.linkedin.com/company/pulumi/" \
 		--exclude "https://marketplace.visualstudio.com/items?itemName=pulumi.build-and-release-task"
 
 .PHONY: validate
 validate:
-	bundle exec jekyll serve --strict_front_matter --host 0.0.0.0 --incremental --detach >/dev/null 2>&1
+	hugo server -D --disableBrowserError --disableLiveReload &>/dev/null &
+	while ! nc -z localhost 1313; do sleep 0.1; done
```

Comment:
It takes a little bit of time for the `hugo server` to start up, so this is to wait until port 1313 is open before proceeding with the broken link validation. I'm not sure if `nc` is available in Travis. We'll see.